### PR TITLE
(Feat) UI: Add Profiler Tab as per new mock

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

- UI: Add Profiler Tab as per new mock
- Added unit test for all the new components with almost 90% coverage

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :

https://user-images.githubusercontent.com/71748675/183733658-6663c6ec-0377-41c2-b6e0-b594d2cf4c2b.mov

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ui

### CODE REVIEW

**Comment by sureshms** [comment]
@ShaileshParmar11 the profile sample slide bar should show the profile percentage the user chose with a number. Cc @harshach

File: openmetadata-ui/src/main/resources/ui/src/axiosAPIs/tableAPI.ts [hunk 1]
```diff
@@ -212,3 +216,27 @@ export const deleteColumnTestCase = (
     configOptions
   );
 };
+
+export const getTableProfilerConfig = async (tableId: string) => {
+  const response = await APIClient.get<string, AxiosResponse<Table>>(
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/axiosAPIs/tableAPI.ts:221]:
```suggestion
  const response = await APIClient.get<Table>(
```

File: openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/TableProfilerV1.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+      },
+      {
+        title: 'Failed',
+        value: formTwoDigitNmber(0),
+        className: 'failed',
+      },
+    ];
+  }, [tableProfile]);
+
+  return !isUndefined(tableProfile) ? (
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/TableProfilerV1.tsx:72]: Change it to positive condition instead

File: openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/mockTableData.ts [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+  updatedAt: 1659764891329,
+  updatedBy: 'anonymous',
+  href: 'http://localhost:8585/api/v1/tables/cb726d24-774b-4603-8ec8-1975760ac2f8',
+  tableType: 'Regular',
+  columns: [
+    {
+      name: 'address_id',
+      dataType: 'NUMERIC',
+      dataTypeDisplay: 'numeric',
+      description: 'Unique identifier for the address.',
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/mockTableData.ts:33]: move this file to `src/mocks`

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**